### PR TITLE
Css updt

### PR DESCRIPTION
**Pull request by damianodamiani**
css colors (and the website?)

### CODE REVIEW

File: cfp.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Call for Papers
```
Comment (duncdrum) [inline cfp.md]:
```suggestion
title: 'Call for Papers'
```

**Review by duncdrum — CHANGES_REQUESTED** [review]
I would put `"` marks around strings which contain ` ` whitespaces

**Comment by damianodamiani** [comment]
ok, now it builds locally -- let me know if all is fine! @duncdrum

**Comment by duncdrum** [comment]
you need to resolve conflicts

**Comment by damianodamiani** [comment]
done, sorry didnt see it before @duncdrum

**Review by duncdrum — APPROVED** [review]

**Pull request merged**